platform/core/PolyPodCoreSwift/Sources/PolyPodCoreSwift/FBObject.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import PolyPodCore
+import FlatBuffers
+
+@dynamicMemberLookup
+/// A wrapper class used to manage the lifetime of the Flatbuffer memory.
+/// It will make sure to deallocate the memory once the model is no more used.
+public final class FBObject<FBModel> {
```

Comment:
Naming nit: in our context, FB reminds us too much of Facebook, not flatbuffers. If you can think about anything better, (`flatb`? `♭`? Can you use unicode in Swift names? 

> I answer myself: yes, you can, but `♭` can be used only for operators, so maybe we could use `ß`?